lib/workers/repository/process/lookup/index.spec.ts

```diff
@@ -1979,5 +1985,104 @@ describe('workers/repository/process/lookup/index', () => {
         },
       ]);
     });
+
+    describe('handles merge confidence', () => {
+      const getMergeConfidenceSpy = jest.spyOn(
+        McApi,
+        'getMergeConfidenceLevel'
+      );
+      const apiBaseUrl = 'https://www.baseurl.com/';
+      const envOrg: NodeJS.ProcessEnv = process.env;
+      const hostRule: HostRule = {
+        hostType: 'merge-confidence',
+        token: 'some-token',
+      };
+
+      beforeEach(() => {
+        process.env = {
+          ...envOrg,
+          RENOVATE_X_MERGE_CONFIDENCE_API_BASE_URL: apiBaseUrl,
+        };
+        hostRules.add(hostRule);
+        initConfig();
+        memCache.reset();
+      });
+
+      afterEach(() => {
+        process.env = envOrg;
+        hostRules.clear();
+        resetConfig();
+      });
+
+      it('gets a merge confidence level for a given update when corresponding packageRule is in use', async () => {
+        const datasource = NpmDatasource.id;
+        const packageName = 'webpack';
+        const newVersion = '3.8.1';
+        const currentValue = '3.7.0';
+        config.packageRules = [{ matchConfidence: ['high'] }];
+        config.currentValue = currentValue;
+        config.packageName = packageName;
+        config.datasource = datasource;
+        httpMock
+          .scope('https://registry.npmjs.org')
+          .get('/webpack')
+          .reply(200, webpackJson);
+        httpMock
+          .scope(apiBaseUrl)
+          .get(
+            `/api/mc/json/${datasource}/${packageName}/${currentValue}/${newVersion}`
+          )
+          .reply(200, { confidence: 'high' });
+
+        const lookupUpdates = (await lookup.lookupUpdates(config)).updates;
+
+        expect(lookupUpdates).toMatchObject([
+          {
+            mergeConfidenceLevel: `high`,
+          },
+        ]);
+      });
+
+      it('doesnt get a merge confidence level when no packageRule is set', async () => {
+        config.currentValue = '3.7.0';
+        config.packageName = 'webpack';
+        config.datasource = NpmDatasource.id;
+        httpMock
+          .scope('https://registry.npmjs.org')
+          .get('/webpack')
+          .reply(200, webpackJson);
+
+        const lookupUpdates = (await lookup.lookupUpdates(config)).updates;
+
+        expect(getMergeConfidenceSpy).toHaveBeenCalledTimes(0);
+        expect(lookupUpdates).not.toMatchObject([
+          {
+            mergeConfidenceLevel: expect.anything(),
+          },
+        ]);
+      });
+
+      it('doesnt set merge confidence value when api is not in use', async () => {
```

Comment:
```suggestion
      it('doesn't set merge confidence value when API is not in use', async () => {
```

Or maybe you to to escape the `'` in `doesn't`?